js/app-super-resolution/README.md

```diff
@@ -0,0 +1,59 @@
+# RUNNING SUPER-RESOLUTION ON IMAGES ACROSS VARIOUS PLATFORMS USING ONNX RUNTIME MODELS
```

Comment:
Let's make it clear here that the app runs on web and android and not iOS yet.

And if you know, the steps needed to get it working on iOS

Reply:
👍